### ISSUE TITLE
Rounding on Reportingstatus page

### ISSUE THREAD

**Issue by MoBosse** [open-sdg/open-sdg#300]
The rounding of the percentages on the rerortingstatus-page does not work correctly (rounds 150/244 to 62%).

Apart from that, in some cases (e.g. 104 _Reported online_ (~43%), 104 _Statistics in progress_ (~43%) and 36 _Exploring data sources_ (~15%)) the bar would have a total width of more (or less) than 100% even with correct rounding.

See: [NRP Germany](https://g205sdgs.github.io/sdg-indicators/en/reporting-status/)

**Comment by LucyGwilliamAdmin** [comment]
For the second part of that it looks like the part of the bar for N/A has gone below for some reason:
![image](https://user-images.githubusercontent.com/43404358/63084484-2ff7e500-bf43-11e9-9fab-8ff719119474.png)

It looks like the problem is caused when the total reporting status totals over 100% as also case with goal 10 (total is 101%):
![image](https://user-images.githubusercontent.com/43404358/63084718-bb717600-bf43-11e9-8e54-1f9d07764d0d.png)

**Comment by brockfanning** [comment]
We've struggled with this for a while, and I think it seems clear that our problem is we're trying to use the same number for:
1. The human-readable percentage
2. The CSS width percentage

I suspect we'll need to start using some rounded number for the human-readable percentage, and a raw non-rounded number for the CSS width percentage.

**Comment by angelamhkim** [comment]
@brockfanning is there an update on how to fix this issue? Or if you have tips on where to get started, our programmer intern could take a shot at solving this.

**Comment by brockfanning** [comment]
@angelamhkim I think it will be a two-part fix.
1. In sdg-build, I was thinking maybe we could add a new item under [here](https://github.com/open-sdg/sdg-build/blob/master/sdg/stats.py#L54) called "percentage_precise". This could be the same as "percentage" but without the rounding.
2. In open-sdg, in reportingstatus.html where the width is set to the percentage ([here](https://github.com/open-sdg/open-sdg/blob/master/_layouts/reportingstatus.html#L28) and [here](https://github.com/open-sdg/open-sdg/blob/master/_layouts/reportingstatus.html#L66)) we could change it from `style="width:{{ status.percentage }}%"` to `style="width:{{ status.percentage_precise }}%"`.

**Comment by lindamejia** [comment]
This was my solution to this:
1. In the open-sdg-site-starter I created the item "percentage_precise" in the the reportingstatus.html, using Liquid instead, for both the [overall status bar](https://github.com/dawncomer/open-sdg-site-starter/blob/c6cb7bee364bdea9fc04c413834424e8de5f12d9/_layouts/reportingstatus.html#L31) and for [status by goal bar](https://github.com/dawncomer/open-sdg-site-starter/blob/c6cb7bee364bdea9fc04c413834424e8de5f12d9/_layouts/reportingstatus.html#L76)
2. Overrode css to get rid of inherent span tag space [goal-stats](https://github.com/dawncomer/open-sdg-site-starter/blob/c6cb7bee364bdea9fc04c413834424e8de5f12d9/assets/css/custom.scss#L26)

**Comment by MoBosse** [comment]
Great. Works fine.
Thanks

**Comment by brockfanning** [comment]
@lindamejia We'd love to get your solution into the platform. Do you have the bandwidth to post it as a PR?

**Comment by lindamejia** [comment]
Hi @brockfanning , I've forked and cloned this repo to create the PR but when I run "make serve" I get this error: 
```
...
cp -r -t site-starter/ tests/site/*
cp: illegal option -- t
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file ... target_directory
make: *** [test.prep] Error 64 
```
Do you know what could be happening?

**Comment by brockfanning** [comment]
I think our Makefile isn't working in all environments. It has probably only been tested on Linux so far. Are you on a Mac maybe? Regardless, I'll look into rewriting the Makefile without using the `-t` option.

**Comment by brockfanning** [comment]
@lindamejia I just tweaked the Makefile - can you pull the latest code and give it another shot?

**Comment by lindamejia** [comment]
That was it! (I'm using a mac) I just created the [PR ](https://github.com/open-sdg/open-sdg/pull/485)

**Comment by brockfanning** [comment]
Fixed by #485 :+1:
